src/components/Components/FieldsTitle.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const FieldsTitle = ({ title }) => (
```

Comment:
If I understand well, `FieldsTitle` here is a title for a group a fields. Maybe `FieldsetTitle` could be more accurate ? What do you think ?

Reply:
Well, yeah. I honestly just took the previous className and transformed it into a component's name so it can be less confusing. But your comment is legit.